mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/TiltakstypeRoutes.kt

```diff
@@ -58,6 +58,10 @@ fun Route.tiltakstypeRoutes(migrerteTiltak: List<Tiltakskode>) {
         get("migrerte") {
             call.respond(migrerteTiltak.map { toArenaKode(it) })
         }
+
+        get("stotterPameldingIModia") {
+            call.respond(pameldingIModia.map { toArenaKode(it) })
```

Comment:
Ah, ser jeg har misforstått :sweat_smile:  Men hvorfor ikke bare benytte tiltakskode i stedet for arenakode?

Reply:
Jeg bare tok samme utgangspunkt som vi allerede har, men jeg kan se på å endre til tiltakskode hvis det ikke er for mye hassle

Reply:
Tror grunnen til at de andre er arenaKode var fordi vi enda ikke hadde fått på plass vår nye `tiltakskode` når det arbeidet ble påstartert :thinking:

Reply:
Ja, det stemmer nok.
Jeg lager et nytt kort som går på å bytte ut arenakode med tiltakskode. Så kan denne PRen være som den er. Det blir litt ryddigere tenker jeg.

Reply:
All right. Tenker i hvert fall det er ryddig å gjøre konfigurasjon på våre tiltakskoder i stedet for arenakoder så langt det lar seg gjøre :+1:

Reply:
Enig!